.babelrc

```diff
@@ -1,3 +1,8 @@
 {
-    "presets" : ["@babel/preset-env"]
+  "presets": ["@babel/preset-env"],
```

Comment:
@frankly034  liaise with @Eazybee on how he managed to solve this. I was expecting to see a .editorconfig file 🤔